org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/handlers/WorkspaceDiagnosticsHandlerTest.java

```diff
@@ -79,6 +79,13 @@ public void setup() throws Exception {
 		handler.addResourceChangeListener();
 	}
 
+	@After
+	@Override
+	public void cleanUp() throws Exception {
+		super.cleanUp();
+		handler.removeResourceChangeListener();
+	}
+
```

Comment:
In the end of this file, there is remoceResourceChangeListener. Looks like it's not needed any more. Right?
```java
	@After
	public void removeResourceChangeListener() {
		handler.removeResourceChangeListener();
	}
```

Reply:
correct